qa/src/test/java/io/camunda/migrator/qa/runtime/variables/VariablesTest.java

```diff
@@ -84,36 +94,50 @@ public void shouldSetPrimitiveVariables() {
         .hasVariable("stringVar", "myStringVar")
         .hasVariable("booleanVar", true)
         .hasVariable("integerVar", 1234)
-        .hasVariable("floatVar", 1.5)
         .hasVariable("doubleVar", 1.5d)
-        .hasVariable("shortVar", (short) 1)
-        .hasVariable("byteVar", (byte) 1)
-        .hasVariable("charVar", (char) 1);
+        .hasVariable("shortVar", (short) 1);
```

Comment:
I think there's no testing for Long variables in this class, I realise the handling is the same for all numbers but can we add one here just for completion?